ShippoTesting/TrackTest.cs

```diff
@@ -24,12 +24,12 @@ public void TestValidGetStatus ()
             Assert.IsNotNull (track.TrackingHistory);
         }
 
-        [Test ()]
-        [ExpectedException (typeof (ShippoException))]
-        public void TestInvalidGetStatus ()
-        {
-            getAPIResource ().RetrieveTracking (CARRIER, "INVALID_ID");
-        }
+        //[Test ()]
```

Comment:
The exception testing was out of date so I had commented it but I then forgot to put it back. I've updated this as well as the other removed test in batch test